### ISSUE TITLE
Invalid meta tag on CMS sites

### ISSUE THREAD

**Issue by mfairchild365** [unlcms/UNL-CMS#920]
From the validator: https://validator.w3.org/nu/?doc=http%3A%2F%2Fwdn.unl.edu%2F

```
A document must not include both a meta element with an http-equiv attribute whose value is content-type, and a meta element with a charset attribute.

<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
```

**Comment by ericras** [comment]
Fixed with e9566f7b5ad8d00bf7ad209c48e9bb4bf9bffd17

**Comment by ericras** [comment]
Moved to avoid hitting the return first: https://github.com/unlcms/unl_fourone/commit/7e39300c87078ced85833eaa054d223967ab3a83
